Logger/FileLogger.cs

```diff
@@ -0,0 +1,23 @@
+using System.IO;
+using System;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string path;
+
+        public FileLogger(string path)
+        {
+            this.path = path;
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            DateTimeOffset dt = DateTimeOffset.Now;
+            StreamWriter file = new StreamWriter(this.path, append: true);
+            file.WriteLine(dt.ToString("g") + " " + this.ClassName + " " + logLevel + ": " + message);
```

Comment:
```suggestion
            file.WriteLine(dt.ToString("g") + " " + ClassName + " " + logLevel + ": " + message);
```
Consider using string interpolation here, rather than appending strings together.